plc4j/integrations/apache-kafka/README.md

```diff
@@ -1,82 +1,190 @@
+PLC4X or from the latest snapshot from https://github.com/apache/plc4x[github].
+//They can also be downloaded from the confluent https://www.confluent.io/hub/[hub].
+
+## Introduction
+
+A connect worker is basically a producer or consumer process with a standard api that Kafka can use to manage it. It is
+able to be run in two modes:-
 
-  Licensed to the Apache Software Foundation (ASF) under one or more
-  contributor license agreements.  See the NOTICE file distributed with
-  this work for additional information regarding copyright ownership.
-  The ASF licenses this file to You under the Apache License, Version 2.0
-  (the "License"); you may not use this file except in compliance with
-  the License.  You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
-
--->
-
-# Kafka Connect PLC4X Connector
-
-The PLC4X Connector streams data from any device accessible through the PLC4X interface.
-
-## Source Connector
-
-See `config/source.properties` for example configuration.
-
-## Quickstart
-
-A Kafka Connect worker can be run in two modes: 
 - Standalone
 - Distributed
 
-Both modes require a Kafka Broker instance to be available.
-Kafka Connect is part of the Kafka distribution. 
+Standalone allows you to run the connector locally from the command line without having to install the jar file on your
+Kafka brokers.
+In distributed mode the connector runs on the Kafka brokers, which requires you to install the jar file on all of your
+brokers. It allows the worker to be distrubuted across the Kafka brokers to provide redundancy and load balancing.
+
+## Quickstart
 
 In order to start a Kafka Connect system the following steps have to be performed:
 
-1) Download the latest version of Apache Kafka binaries from here: https://kafka.apache.org/downloads
+1) Download the latest version of Apache Kafka binaries from here: https://kafka.apache.org/downloads.
+
 2) Unpack the archive.
-3) Copy the `target/plc4j-apache-kafka-0.7.0-SNAPSHOT-uber-jar.jar` to the Kafka `libs` directory.
-4) Copy the files in the `config` to Kafka's `configs` directory.
+
+3) Copy the `target/plc4j-apache-kafka-0.8.0-uber-jar.jar` to the Kafka `libs` or plugin directory specified
+in the config/connect-distributed.properties file.
+
+4) Copy the files in the `config` to Kafka's `config` directory.
 
 ### Start a Kafka Broker
 
 1) Open 4 console windows and change directory into that directory
-2) Start Zookeeper: 
-        
-        bin/zookeeper-server-start.sh config/zookeeper.properties 
+2) Start Zookeeper:
+
+        bin/zookeeper-server-start.sh config/zookeeper.properties
```

Comment:
I even heard that new versions of Kafka work without Zookeeper ... we should check this and perhaps update this section later on